### ISSUE TITLE
Why is the user script so large and its development version so small?

### ISSUE THREAD

**Issue by hongyi-zhao** [dotennin/baidu-pan-downloader#35]
As noted in the README [here](https://github.com/dotennin/baidu-pan-downloader#this-repository-contains-a-stub-project-for-developing-usercript-extensions-with-reactreduxtoolkittypescript):

```
In Firefox or Chrome, install Greasemonkey or Tampermonkey.
You can then test the userscript by installing it here. Navigate to google 
and you should see a react component inserted at the bottom of the page.
```
The [userscript](https://github.com/siefkenj/react-userscripts/raw/master/dist/react-userscripts.user.js) used for testing is so huge: there are 9680 lines in it. Wile I also noticed that the corresponding development version of this script, i.e., the scripts located at [here](https://github.com/dotennin/baidu-pan-downloader/blob/master/dist/react-userscripts-dev.user.js) and [here](https://github.com/dotennin/baidu-pan-downloader/blob/master/dist/react-userscripts-dev.user.js), are so small.

To summarize, my question is: Why is the user script so large and its development version so small?

Regards,
HZ

**Comment by dotennin** [comment]
> To summarize, my question is: Why is the user script so large and its development version so small?
Please see : https://github.com/dotennin/baidu-pan-downloader#development-and-dynamic-loading
